thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/client/cache/CollectionMaxDataTimeCacheLoader.java

```diff
@@ -64,6 +64,10 @@ public Long load(String collection) throws Exception {
       if (zkSegmentDataList != null && !zkSegmentDataList.isEmpty()) {
         for (ZNRecord record : zkSegmentDataList) {
           try {
+            if ("IN_PROGRESS".equals(record.getSimpleField("segment.realtime.status"))) {
+              maxTime =System.currentTimeMillis();
```

Comment:
formatting is off. Its probably better to get maxTime from Pinot every X minutes?